views/forms/UserProfileForm/UserProfileImage.css

```diff
@@ -0,0 +1,15 @@
+.UserProfileForm_profileImage {
+  display: flex;
+  align-items: center;
+  padding: 24px 0px;
+}
+.UserProfileFormFileUploadInput {
+  display: none;
+}
+.UserProfileFormAlert {
+  padding-right: 506px;
```

Comment:
what is this for?

Reply:
The alert component takes up the entire width of it's parent, the parent is 100% width but in the designs the alert component takes about 40% of the screen, and to not set the width explicitly the padding makes it compact

Reply:
we can specify a max-width in this case as well, I believe.

Reply:
[6792e0c](https://github.com/AMP-SCZ/dpdash/pull/478/commits/6792e0c50a30c6288c1f3db796fdf2ea55f5d38f)